src/components/page-header/page-header.component.html

```diff
@@ -3,7 +3,7 @@
     <div *ngIf="!condensed" class="page-header-content">
 
         <!-- Logo/product acronym -->
-        <div uxFocusIndicator tabindex="0" class="page-header-logo-container" role="presentation" [style.backgroundColor]="logoBackground" [style.color]="logoForeground" (click)="logoClick.emit($event)">
+        <div uxFocusIndicator (keydown.enter)="logoClick.emit($event)" [keyboardFocusIndicator]="hasLogoClick" [attr.tabindex]="hasLogoClick ? 0 : -1" [class.clickable]="hasLogoClick" class="page-header-logo-container" role="presentation" [style.backgroundColor]="logoBackground" [style.color]="logoForeground" (click)="logoClick.emit($event)">
```

Comment:
Do we need `[keyboardFocusIndicator]="hasLogoClick"`? I would have assumed if the item has a tab index of `-1` when it is not tabbable then it would not get focused? I could be wrong, just checking.

Reply:
Removed <code>[keyboardFocusIndicator]="hasLogoClick"</code>, checked and it isn't needed.